.github/workflows/macos.yml

```diff
@@ -71,9 +84,38 @@ jobs:
 
       - run: make
 
+      # Launchable CLI requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up Java
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Set up Launchable
+        id: setup_launchable
+        run: |
+          set -ex
+          pip install launchable
+          launchable verify
+          : # The build name cannot include a slash, so we replace the string here.
+          github_ref=`echo ${{github.ref}} | sed 's/\//_/g'`
```

Comment:
A small thing:

```suggestion
          github_ref=`echo ${{ github.ref }} | sed 's/\//_/g'`
```

Reply:
One tip to implement for such kind of issue. You can take a look and observe the existing code in ruby/ruby, when you add something.

Reply:
Thanks. Updated

Reply:
Thanks. I confirmed it on the latest PR.